src/Avalonia.Themes.Fluent/Controls/FluentControls.xaml

```diff
@@ -2,6 +2,11 @@
     xmlns="https://github.com/avaloniaui"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     x:ClassModifier="internal">
+
+  <Style Selector="HyperlinkButton TextBlock">
+    <Setter Property="TextDecorations" Value="Underline"/>
+  </Style>
```

Comment:
This is not good and I consider it a work-around. However, I'm not sure what else to do here.

Attempting the strategy noted here: https://github.com/AvaloniaUI/Avalonia/issues/7640#issuecomment-1043865206. Doesn't work after control themes. @maxkatz6 Did you come up with a work-around or different strategy for this?

Reply:
Yeah, it's not ideal. But I was going to remove these styles at all, keeping only resource dictionaries.

Reply:
Will take a look at how we can avoid that

Reply:
@robloo TextDecorationsProperty is inherit=true property. You should be able to set `Property="TextBlock.TextDecorations" Value="True"` on HyperlinkButton  styles. I haven't double checked it, but should just work.

Reply:
> @maxkatz6 Did you come up with a work-around or different strategy for this?

I don't work on that project anymore, so who knows.

Reply:
> Property="TextBlock.TextDecorations" Value="True"

Yea, just setting the attached property works. I could have sworn I tried that already but guess I made a mistake.